tools/packages/debian/build-deb.sh

```diff
@@ -2,11 +2,11 @@
 # Should be run from the root of the source tree
 # Set env var REVISION to overwrite the 'revision' field in version string
 
-if [ ! -d debian ]; then
+if [ ! -d tools/packages/debian ]; then
```

Comment:
Nit: how about using a variable to hold the location of the Debian packaging scripts, and use that later:

PKG_SCRIPTS_DIR=tools/packages/debian